### PR TITLE
Forward port release note changes from 1.4

### PR DESCRIPTION
**Pull request by timcharper**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -109,32 +109,43 @@ All validation specified in the RAML is now programatically enforced, leading to
...
 **Note**: Instantly means as soon as marathon becomes aware of the unreachable task. By default Marathon is notified after 75 seconds by Mesos
-  that an agent is disconnected. You can change this duration in Mesos by configuring `agent-ping-timeout` and `max_agent_ping_timeouts`.
+  that an agent is disconnected. You can change this duration in Mesos by configuring `agent_ping_timeout` and `max_agent_ping_timeouts`.
 
 #### Migrating unreachableStrategy
+
 If you want all of your apps and pods to adopt a `UnreachableStrategy` that retains the previous behavior where instance were immediately replaced so that you does not have to update every single app definition.
 
-To change the `unreachableStrategy` of all apps and pods, you can define the environment parameter `MIGRATION_1_4_6_UNREACHABLE_STRATEGY` which leads to the following behavior during migration:
+To change the `unreachableStrategy` of all apps and pods, you may set the environment variable `MIGRATION_1_4_6_UNREACHABLE_STRATEGY` to `true`, which leads to the following behavior during migration:
```
Comment (sascala) [inline changelog.md]: suggest removing "you may".

File: changelog.md [hunk 2]
```diff
@@ -109,32 +109,43 @@ All validation specified in the RAML is now programatically enforced, leading to
...
 
 When opting in to the unreachable migration step
 1) all app and pod definitions that had a config of `UnreachableStrategy(300 seconds, 600 seconds)` (previous default) are migrated to have `UnreachableStrategy(0 seconds, 0 seconds)`
 2) all app and pod definitions that had a config of `UnreachableStrategy(1 second, x seconds)` are migrated to have `UnreachableStrategy(0 seconds, x seconds)`
 3) all app and pod definitions that had a config of `UnreachableStrategy(1 second, 2 seconds)` are migrated to have `UnreachableStrategy(0 seconds, 0 seconds)`
 
+Please note that if you set this variable after upgrading to 1.4.6 it will have no effect. Also, the `UnreachableStrategy` default has not been changed, so apps and pods must specify the 0, 0 unreachableStrategy JSON as seen above.
```
Comment (sascala) [inline changelog.md]: **Note:** If you set this variable...

File: changelog.md [hunk 3]
```diff
@@ -109,32 +109,43 @@ All validation specified in the RAML is now programatically enforced, leading to
...
+  "unreachableStrategy": {
     "inactiveAfterSeconds": 0,
-    "expunceAfterSeconds": 0
+    "expungeAfterSeconds": 0
+  },
+  ...
 }
 ```
+
 **Note**: Instantly means as soon as marathon becomes aware of the unreachable task. By default Marathon is notified after 75 seconds by Mesos
```
Comment (sascala) [inline changelog.md]: ^ comma after "default"

File: changelog.md (hunk 1 above)
Comment (timcharper) [inline changelog.md]: ✅

File: changelog.md (hunk 2 above)
Comment (timcharper) [inline changelog.md]: ✅

File: changelog.md (hunk 3 above)
Comment (timcharper) [inline changelog.md]: ✅

**Pull request merged**